service/storage_service.cc

```diff
@@ -2462,6 +2462,9 @@ future<> storage_service::removenode(locator::host_id host_id, std::list<locator
                     });
```

Comment:
Re commit message:

1. please use "test/topology:" or "test.py" as the prefix
2. please use a grammatically correct English statement instead of "removenode fail finish group0 remove"